Rakefile

```diff
@@ -6,3 +6,9 @@ Rake::TestTask.new do |t|
   t.libs << "test"
   t.pattern = "test/**/*_test.rb"
 end
+
+task :test2
```

Comment:
Can we give this more meaningful name?

Reply:
Also when I execute the command `bundle exec rake test2` nothing happens. Could you please check that.

Reply:
Done 👍